cli/popper/scm.py

```diff
@@ -1,30 +1,40 @@
 import os
-import popper.utils as pu
+import git
+import shutil
+import click
+import sys
+
+try:
+    repo = git.Repo(search_parent_directories=True)
+except git.exc.InvalidGitRepositoryError:
+    click.secho(
+        'ERROR: Unable to find root of project. Initialize repository first.',
+        fg='red', bold=True, err=True, nl=False
+    )
+    sys.exit(1)
```

Comment:
let's use the `utils.fail()` function instead of directly calling `click`

Reply:
Same

Reply:
@ivotron  Here if i am doing 
```
try:
    repo = git.Repo(search_parent_directories=True)
except git.exc.InvalidGitRepositoryError:
    from popper.utils import fail
    fail('Unable to find root of project. Initialize repository first.')
```
When i am running `popper` in a directory where git is not initialized error comes up

Reply:
Which error? Circular dependencies?

Reply:
yes

Reply:
@ivotron Solved it